apps/src/applab/debugArea.js

```diff
@@ -0,0 +1,236 @@
+/* jshint
+ funcscope: true,
+ newcap: true,
+ nonew: true,
+ shadow: false,
+ unused: true,
+
+ maxlen: 90,
+ maxparams: 3,
+ maxstatements: 200
+ */
+/* global $ */
+
+var applabMsg = require('./locale');
+var dom = require('../dom');
+var utils = require('../utils');
+
+var debugArea = module.exports;
+
+/** @type {jQuery} */
+var rootDiv_ = null;
```

Comment:
I think it's fine to leave it non-object-oriented. I'm in favor of moving to OO in general, but we are not currently making a strategic effort to move in that direction.